plugins/lookup/dig.py

```diff
@@ -70,29 +70,73 @@
     description:
       - list of composed strings or dictonaries with key and value
         If a dictionary, fields shows the keys returned depending on query type
-    fields:
-       ALL: owner, ttl, type
-       A: address
-       AAAA: address
-       CNAME: target
-       DNAME: target
-       DLV: algorithm, digest_type, key_tag, digest
-       DNSKEY: flags, algorithm, protocol, key
-       DS: algorithm, digest_type, key_tag, digest
-       HINFO:  cpu, os
-       LOC: latitude, longitude, altitude, size, horizontal_precision, vertical_precision
-       MX: preference, exchange
-       NAPTR: order, preference, flags, service, regexp, replacement
-       NS: target
-       NSEC3PARAM: algorithm, flags, iterations, salt
-       PTR: target
-       RP: mbox, txt
-       SOA: mname, rname, serial, refresh, retry, expire, minimum
-       SPF: strings
-       SRV: priority, weight, port, target
-       SSHFP: algorithm, fp_type, fingerprint
-       TLSA: usage, selector, mtype, cert
-       TXT: strings
+    contains:
+       ALL:
+           description:
+               - owner, ttl, type
```

Comment:
I have no idea whether this change is correct (since I find the doc very hard to understand), but at least it is not just some random structured data anymore :)